### DISCUSSION TITLE
Is there any way to rename labels in side bar (in admin UI)

### DISCUSSION THREAD

**Discussion by lakshmajee** [vendurehq/vendure#1815] [Q&A]
Is there any way to rename Sidebar labels for something else? 
Example `Collection` to `Categories` and `Facets` to `tags` or `filters` in the admin dashboard using `vendure.config.ts` or similar configs ?

![sample](https://user-images.githubusercontent.com/15211916/193342089-d6537d3f-bec7-4999-ab24-694133bac1d7.png)

**Comment by michaelbromley** [comment]
Hi,
I think this should be possible by creating a custom UI build and supplying translations that override those defined in the default translation file. See https://www.vendure.io/docs/plugins/extending-the-admin-ui/adding-ui-translations/

**Reply by lakshmajee** [reply]
got it, Thanks

**Reply by lakshmajee** [reply]
Okay, `translations` are the only way to do this. Is there any alternative ? like mapping keys in `vendure.config.ts` ?

**Reply by michaelbromley** [reply]
No, this is not determined by anything in `vendure-config.ts`.
